utxonursery.go

```diff
@@ -338,3 +695,231 @@ func newSweepPkScript(wallet lnwallet.WalletController) ([]byte, error) {
 
 	return txscript.PayToAddrScript(sweepAddr)
 }
+
+// addSerializedKidToList is used to store outputs to disk because multiple outputs
+// may need to be swept for a given block height.
+func addSerializedKidToList(w io.Writer, existingOutputs []byte, kid *kidOutput) error {
+	var b bytes.Buffer
+
+	if err := serializeKidOutput(&b, kid); err != nil {
```

Comment:
The new `kidOutput ` can be directly written into the passed `io.Writer` without any need for an intermediate `bytes.Buffer: 
```
return serializeKidOutput(w, kid)
```